cachyos-keyring/PKGBUILD

```diff
@@ -1,24 +1,26 @@
 pkgname=cachyos-keyring
-pkgver=3
+pkgver=20240331
 pkgrel=1
 pkgdesc="CachyOS keyring"
 arch=(any)
 url="https://raw.githubusercontent.com/CachyOS/CachyOS-PKGBUILDS/master/$pkgname"
 license=('GPL3')
 groups=(cachyos)
 install=$pkgname.install
-[ -r $install ] || curl -o $install -s --max-time 30 $url/$install
-
 source=(
+  $url/Makefile
```

Comment:
welp why we fetch from net here

Reply:
I just kept it as it was before, we can remove that